.travis.yml

```diff
@@ -8,16 +8,16 @@ before_install:
 script: bundle exec rake
 
 rvm:
-  - 2.4.9
-  - 2.5.7
-  - 2.6.5
-  - 2.7.0
+  - 2.4.10
```

Comment:
I'm oretty sure you can set `2.4` to automatically get the last available version.